test/functional/test_runner.py

```diff
@@ -224,8 +224,12 @@
     'rpc_scantxoutset.py'
 ]
 
+USDT_SCRIPTS = [
+    'interface_usdt_net.py',
+]
+
 # Place EXTENDED_SCRIPTS first since it has the 3 longest running tests
-ALL_SCRIPTS = EXTENDED_SCRIPTS + BASE_SCRIPTS
+ALL_SCRIPTS = EXTENDED_SCRIPTS + BASE_SCRIPTS + USDT_SCRIPTS
```

Comment:
In Bitcoin, `interface_usdt_net.py` seems to be added `BASE_SCRIPTS`.

Reply:
Yes, bitcoin has support in the test framework to check if a module is present and skip tests if it is missing. I am disabling these tests as they do not complete successfully.